resources/provider/migrations/postgres/22_v0.12.0.down.sql

```diff
@@ -0,0 +1,7 @@
+-- Autogenerated by migration tool on 2022-03-17 09:22:40
+-- CHANGEME: Verify or edit this file before proceeding
+
+-- Resource: ec2.nat_gateways
+ALTER TABLE aws_ec2_nat_gateway_addresses ALTER COLUMN allocation_id SET NOT NULL;
```

Comment:
Had to add this manually. Can this be added by the migration tool, @disq

Reply:
No, migration tool doesn't support Nulls/Uniques.